blog/src/main/kotlin/com/example/blog/domain/post/domain/Post.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.blog.domain.post.domain
+
+import com.example.blog.domain.category.domain.Category
+import com.example.blog.domain.content.domain.Content
+import com.example.blog.domain.postLocation.domain.PostLocation
+import com.example.blog.domain.series.domain.Series
+import jakarta.persistence.*
+import lombok.AllArgsConstructor
+import lombok.NoArgsConstructor
+
+import java.util.Date
+import java.util.UUID
+
+@Entity
+@Table(name = "posts")
```

Comment:
```suggestion
@Entity(name = '~')
```
 Table어노테이션 사용해 table 이름 정하는것보다
이렇게 entity어노테이션을 사용해 table 이름 정하는게 좋을 것 같아요

Reply:
수정하였습니다

Reply:
5fcfe47035439c91c42834206a153e3b807f303b

Reply:
fea81d7d726fa45b9bdde89a39ad097a12f60ada